README.md

```diff
@@ -69,7 +69,7 @@ A single `*.csv` file containing data similar to:
 #### Run the job
 
 ```bash
-INPUT_FILE_PATH="src/test/resources/data/words.txt" JOB=thoughtworks.wordcount.WordCount ./batect run-job
+INPUT_FILE_PATH="src/test/resources/data/words.txt" OUTPUT_PATH="output" JOB=thoughtworks.wordcount.WordCount ./batect run-job
```

Comment:
Can we make it implicit?

Reply:
It was implicit earlier. However because the output of one stage was the input for the other, the implicit path didn't work. And we had to call this out explicitly.